server/search.go

```diff
@@ -38,6 +38,7 @@ type SearchResult struct {
 	Redirect string             `json:"r,omitempty"`
 	HasMore  bool               `json:"m,omitempty"`
 	Timing   SearchResultTiming `json:"t,omitempty"`
+	ResultCount int64
```

Comment:
Can you add an annotation like the other keys so we can have a shorter name in the json? maybe `json:"c,omitempty"`?